scripts/deployment/phase2-assets/collaterals/deploy_origin_eth_mainnet.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { baseL2Chains, networkConfig } from '../../../../common/configuration'
+import { bn, fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  PRICE_FEEDS,
+  ORACLE_ERROR,
+  ORACLE_TIMEOUT,
+} from '../../../../test/plugins/individual-collateral/origin/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { priceTimeout } from '../../utils'
+import { OETHCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Origin ETH to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy Origin ETH Collateral - wOETH  **************************/
+
+  // Only for Base
+  if (!baseL2Chains.includes(hre.network.name)) {
+    throw new Error(`Unsupported chainId: ${chainId}`)
+  }
+
+  const OETHCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+    'OETHCollateral'
+  )
+
+  const collateral = <OETHCollateral>await OETHCollateralFactory.connect(
+    deployer
+  ).deploy(
+    {
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: PRICE_FEEDS.OETH_ETH, // ETH/OETH
+      oracleError: ORACLE_ERROR.toString(),
+      erc20: networkConfig[chainId].tokens.wOETH,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: 1, // to avoid impact in calculations
+      targetName: hre.ethers.utils.formatBytes32String('ETH'),
+      defaultThreshold: bn(0).toString(), // depeg checks not used
+      delayUntilDefault: bn('86400').toString(), // 24h
+    },
+    fp('1e-4').toString(), // revenueHiding = 0.01%
+    PRICE_FEEDS.ETH_USD, // uoaPerTargetChainlinkFeed
+    ORACLE_TIMEOUT // uoaPerTarget timeout
```

Comment:
this should stay just the oracle timeout associated with the ETH_USD oracle

Reply:
Changed in this commit: [839d80f](https://github.com/reserve-protocol/protocol/pull/1239/commits/839d80fb57ace5ba1470ae9aed555926317a84cc)